admin/nginx-https.conf.template

```diff
@@ -26,16 +26,20 @@ server {
 
     include /etc/nginx/snippets/letsencrypt.conf;
 
+    auth_basic "Restricted";
+    auth_basic_user_file /etc/nginx/.htpasswd;
+
+    # Redirect to repondeur if there is no changelog
+    location = / {
+        alias /srv/zam/;
+        try_files index.html /repondeur/;
+    }
+
     location / {
```

Comment:
Is that entry still relevant?

Reply:
There are some let’s encrypt related files there, I supposed we were supposed to serve them. What do you think?

Reply:
Oh, I missed the equal sign above, my bad.

Reply:
No, nginx config format is a bit esoteric...